package.json

```diff
@@ -6,7 +6,50 @@
     "node": ">=18.0.0"
   },
   "type": "module",
-  "main": "./build/index.js",
+  "exports": {
+    "./promise": "./build/common/promise.js",
+
+    "./cache-assets": "./build/client/cache-assets.js",
+
+    "./cors": "./build/server/cors.js",
+    "./get-client-ip-address": "./build/server/get-client-ip-address.js",
+    "./get-headers": "./build/server/get-headers.js",
+    "./is-prefetch": "./build/server/is-prefetch.js",
+    "./json-hash": "./build/server/json-hash.js",
+    "./named-action": "./build/server/named-action.js",
+    "./parse-accept-header": "./build/server/parse-accept-header.js",
+    "./preload-route-assets": "./build/server/preload-route-assets.js",
+    "./respond-to": "./build/server/respond-to.js",
+    "./responses": "./build/server/responses.js",
+    "./rolling-cookie": "./build/server/rolling-cookie.js",
+    "./safe-redirect": "./build/server/safe-redirect.js",
+    "./typed-cookie": "./build/server/typed-cookie.js",
+    "./typed-session": "./build/server/typed-session.js",
+
+    "./client-only": "./build/react/client-only.js",
+    "./external-scripts": "./build/react/external-scripts.js",
+    "./fetcher-type": "./build/react/fetcher-type.js",
+    "./handle-conventions": "./build/react/handle-conventions.js",
+    "./matches-type": "./build/react/matches-type.js",
+    "./server-only": "./build/react/server-only.js",
+    "./use-debounced-fetcher": "./build/react/use-debounced-fetcher.js",
+    "./use-delegated-anchors": "./build/react/use-delegated-anchors.js",
+    "./use-global-pending-state": "./build/react/use-global-pending-state.js",
+    "./use-hydrated": "./build/react/use-hydrated.js",
+    "./use-should-hydrate": "./build/react/use-should-hydrate.js",
+
+    "./sse/server": "./build/server/event-stream.js",
+    "./sse/react": "./build/react/use-event-source.js",
+
+    "./locales/server": "./build/server/get-client-locales.js",
+    "./locales/react": "./build/react/use-locales.js",
+
+    "./honeypot/server": "./build/server/honeypot.js",
+    "./honeypot/react": "./build/react/honeypot.js",
+
+    "./csrf/server": "./build/server/csrf.js",
+    "./csrf/react": "./build/react/csrf.js"
```

Comment:
This one is pointing to a file that doesn't exists, the file is `./build/react/authenticity-token.js`